assets/js/base/components/totals/totals-coupon-code-input/index.js

```diff
@@ -62,18 +67,29 @@ const TotalsCouponCodeInput = ( {
 			>
 				<PanelRow className="wc-block-coupon-code__row">
 					<form className="wc-block-coupon-code__form">
-						<TextInput
-							id={ `wc-block-coupon-code__input-${ instanceId }` }
-							className="wc-block-coupon-code__input"
-							label={ __(
-								'Enter code',
-								'woo-gutenberg-products-block'
-							) }
-							value={ couponValue }
-							onChange={ ( newCouponValue ) =>
-								setCouponValue( newCouponValue )
-							}
-						/>
+						<div>
```

Comment:
needed to add the extra container here so that the validation error would go below the text input for this row.

Reply:
Why can't this be solved with CSS display block? Is it due to flexbox? There are ways to work around that even with flexbox if we want cleaner markup.

That said, from accessibility standpoint we may want to link these elements somehow. Could we pass it down to TextInput to render?

Reply:
That would also let us handle classnames in the TextInput component instead of doing it here.

Reply:
Actually, I would move it below so the validation error spans under the button. This way I think text would not break in two lines.
![imatge](https://user-images.githubusercontent.com/3616980/76755769-bf3cd380-6784-11ea-9d4a-35b7021c2f5f.png)

Reply:
Done.